doc/nrf/ug_thingy91.rst

```diff
@@ -327,3 +306,12 @@ To build and program the source code from the command line, complete the followi
        west flash
 
    #. The device will reset and run the programmed sample or application.
+
+
+.. note::
+
+   When building apps for thingy91, the build system changes the signing algorithm of MCUBoot so that it uses the default RSA Keys.
```

Comment:
```suggestion
   When building apps for Thingy:91, the build system changes the signing algorithm of MCUBoot so that it uses the default RSA Keys.
```